.changelogs/9381.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Update MomentJS dependency to 2.29.2.",
+  "title": "Updated the Moment.js dependency from `2.24.0` to `2.29.3`, improving Handsontable's date format matching.",
```

Comment:
I think I'd rather concentrate on the security aspect of the change, as it's not exactly clear if the change on Moment's side is something we want or not.

Reply:
@jansiegel - sure (I followed [this ](https://handsoncode.slack.com/archives/C4PV4S33Q/p1650384660407479?thread_ts=1649328227.856499&cid=C4PV4S33Q) but took it out of context)

How about this: [a430fda](https://github.com/handsontable/handsontable/pull/9409/commits/a430fda7605f72ad4b29e85361694d52cbb1d27f)

Reply:
> @jansiegel - sure (I followed [this ](https://handsoncode.slack.com/archives/C4PV4S33Q/p1650384660407479?thread_ts=1649328227.856499&cid=C4PV4S33Q) but took it out of context)
> 
> How about this: [a430fda](https://github.com/handsontable/handsontable/pull/9409/commits/a430fda7605f72ad4b29e85361694d52cbb1d27f) (not to sound too alarmist)

Oh, I missed [this](https://handsoncode.slack.com/archives/C4PV4S33Q/p1650384660407479?thread_ts=1649328227.856499&cid=C4PV4S33Q). Still, I'd rather keep the security information in the changelog, and maybe elaborate about the format matching in the release notes/blog post. WDYT?

https://github.com/handsontable/handsontable/pull/9409/commits/a430fda7605f72ad4b29e85361694d52cbb1d27f looks great to me 👍